Adyen/Analytics/AnalyticsProvider/TelemetryTracker.swift

```diff
@@ -35,6 +35,7 @@ public protocol TelemetryTrackerProtocol {
 
 // MARK: - TelemetryTrackerProtocol
 
+/// : nodoc:
 extension AnalyticsProvider: TelemetryTrackerProtocol {
 
     func trackTelemetryEvent(flavor: TelemetryFlavor) {
```

Comment:
Missing access identifier. Should be `internal` ?

Reply:
Done